app/Http/Controllers/ScholarPrePhdSeminarController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\PrePhdSeminar;
+use App\Models\Scholar;
+use App\Models\ScholarAppeal;
+use App\Types\RequestStatus;
+use App\Types\ScholarAppealTypes;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+
+class ScholarPrePhdSeminarController extends Controller
+{
+    public function request(Scholar $scholar)
+    {
+        $this->authorize('apply', [PrePhdSeminar::class, $scholar]);
+
+        return view('research.scholars.pre_phd_form', [
+            'scholar' => $scholar,
+        ]);
+    }
+
+    public function show(Scholar $scholar, PrePhdSeminar $appeal)
+    {
+        $this->authorize('view', [PrePhdSeminar::class, $scholar, $appeal]);
```

Comment:
In this case you can simply pass the instance of PrePhdSeminar and $scholar...

```
authorize('view', [$appeal, $scholar])
```

The policy will be inferred by the first variable..